validation/standard/versions.go

```diff
@@ -52,14 +53,38 @@ type VersionTags struct {
 
 var (
 	Release            Tag
-	NetworkVersions                           = make(map[string]VersionTags)
+	ContractVersions                          = make(map[string]map[Tag]superchain.ContractVersions)
```

Comment:
Can we delete the `VersionTags` type just above?

Reply:
Good call. Done here: https://github.com/ethereum-optimism/superchain-registry/pull/768/commits/f7a0a3a55765aba4dd795dc700938bfc44a46e03